src/components/DashboardEditor/editorUtils.jsx

```diff
@@ -0,0 +1,278 @@
+                unit: '%',
+                label: 'Key 1',
+              },
+              {
+                dataSourceId: 'key2',
+                unit: 'lb',
+                label: 'Key 2',
+              },
+            ],
+          },
+        }
+      : type === CARD_TYPES.TIMESERIES
+      ? {
+          ...baseCardProps,
+          content: {
+            series: [],
+            xLabel: 'Time',
+            yLabel: 'Temperature (˚F)',
+            chartType: TIME_SERIES_TYPES.LINE,
+            includeZeroOnXaxis: true,
+            includeZeroOnYaxis: true,
+            timeDataSourceId: 'timestamp',
+            addSpaceOnEdges: 1,
+          },
+          interval: 'day',
+        }
+      : type === CARD_TYPES.BAR
+      ? {
+          ...baseCardProps,
+          content: {
+            type: BAR_CHART_TYPES.SIMPLE,
+            xLabel: 'Cities',
+            yLabel: 'Total',
+            layout: BAR_CHART_LAYOUTS.VERTICAL,
+            series: [
+              {
+                dataSourceId: 'particles',
+                label: 'Particles',
+              },
+              {
+                dataSourceId: 'temperature',
+                label: 'Temperature',
+              },
+              {
+                dataSourceId: 'emissions',
+                label: 'Emissions',
+              },
+            ],
+            categoryDataSourceId: 'city',
+          },
+        }
+      : type === CARD_TYPES.IMAGE
+      ? {
+          ...baseCardProps,
+          content: {
+            alt: 'landscape.jpg',
+            src: sampleImage,
+            hideMinimap: true,
+            hideHotspots: false,
+            hideZoomControls: false,
+          },
+        }
+      : type === CARD_TYPES.TABLE
+      ? {
+          ...baseCardProps,
+          content: {
+            columns: [
+              {
+                dataSourceId: 'alert',
+                label: 'Alert',
+                priority: 1,
+              },
+              {
+                dataSourceId: 'count',
+                label: 'Count',
+                priority: 3,
+              },
+              {
+                dataSourceId: 'hour',
+                label: 'Hour',
+                priority: 2,
+                type: 'TIMESTAMP',
+              },
+              {
+                dataSourceId: 'pressure',
+                label: 'Pressure',
+                priority: 2,
+              },
+            ],
+            threshold: [
+              {
+                dataSourceId: 'pressure',
+                comparison: '>=',
+                value: 1,
+                severity: 1,
+                label: 'Pressure',
+                showSeverityLabel: true,
+                severityLabel: 'Critical',
+              },
+            ],
+          },
+        }
+      : baseCardProps;
+  return cardData;
+};
+
+export const getCardPreview = (cardData, isSelected, onSelectCard, onRemoveCard) => {
+  const isCardJsonValid = cardJson => {
```

Comment:
this would be more efficient as a switch, especially once the other card types are added